orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Stage.java

```diff
@@ -30,6 +30,7 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.fasterxml.jackson.databind.node.TreeTraversingParser;
+import com.google.common.base.Strings;
```

Comment:
Do we need this dependency?

Reply:
yes, [`Strings.isNullOrEmpty`](https://github.com/spinnaker/orca/pull/3011/files#diff-b9a1511802a5c213abc5e0d0e977f8e6R437) comes from there